dali/server/daldap.cpp

```diff
@@ -144,24 +146,50 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
         Owned<ISecUser> user = ldapsecurity->createUser(username);
         user->credentials().setPassword(password);
 
-        //Check user's digital signature, if present
         bool authenticated = false;
-        if (!isEmptyString(udesc->querySignature()))
+
+        //Check that the digital signature provided by the caller (signature of
+        //caller's "username;timeStamp") matches what we expect it to be
+        if (!isEmptyString(udesc->queryUserTimeStampSignature()))
```

Comment:
for an 'old' 3.14 this will be just 'signature' , not the expected 'userTimeStampSignature'

Reply:
Same as above, 3.14 is new to Master